virttest/data_dir.py

```diff
@@ -214,8 +216,16 @@ def get_tmp_dir(public=True):
 
 
 def get_download_dir():
-    if not os.path.isdir(DOWNLOAD_DIR):
-        shutil.copytree(BASE_DOWNLOAD_DIR, DOWNLOAD_DIR)
+    """
+    Returns the dir that holds download definition files, AKA asset files
+
+    Before retuning the download directory, it makes sure the user, writable
+    copy of the download dir is created, and that all asset definition
+    file are in sync.
+
+    :returns: a writable directory populated with asset definition files
+    """
+    dir_util.copy_tree(BASE_DOWNLOAD_DIR, DOWNLOAD_DIR)
```

Comment:
Actually I see one difference and that is that the user can put his own assets into his read-write location, but he could not add those into the system-wide location...

Well I'd prefer a hierarchy instead (look into avocado-datadir if asset not found look into system-wide place). But as this already exists, we should think on how to avoid deeper problems.

If the purpose of this code is to allow users to store their own asset files, or modified avocado asset files, then your change would overwrite their changes, which is IMO not correct. So I think you should check missing files from there and add only those, otherwise users might be surprised.

Alternatively you can implement the hierarchy, which would IMO be the best.